sdk/src/utils.ts

```diff
@@ -260,3 +260,20 @@ export const assertError = async <T>(fn: Promise<T>, invariantError: InvariantEr
     throw new Error('error wasn not thrown')
   }
 }
+
+export const assertThrowsAsync = async (fn: Promise<any>, word?: string) => {
```

Comment:
Tt would be better to unify this with `assertError`. Play around with different solutions, below are some sample possibilities:
1. common function
2. different naming: `assertQueryError`, `assertTxError`